### PR TITLE
Fix some typos in JavaDocs

### PR DESCRIPTION
**Pull request by andrm**

### CODE REVIEW

**Comment by JerryShea** [comment]
Looks good @andrm thanks for the improvement. For us to merge we will need you to sign the [CA](https://chronicle.software/contributor-agreement/) - can you do this please?

**Comment by andrm** [comment]
Of course, done.

File: src/main/java/net/openhft/chronicle/queue/ExcerptTailer.java [hunk 1]
```diff
@@ -136,8 +136,8 @@ default long lastReadIndex() {
      * current millisecond, toEnd() may not see it, This is because for performance reasons, the
      * queue.lastCycle() is cached, as finding the last cycle is expensive, it requires asking the
      * directory for the Files.list() so, this cache is only refreshed if the call toEnd() is in a
-     * new millisecond. Hence a whole milliseconds with of data could be added to the
-     * chronicle-queue that toEnd() won’t see. For appenders that that are using the same queue
+     * new millisecond. Hence, a whole milliseconds with of data could be added to the
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/queue/ExcerptTailer.java:139]: "with" should be "worth"

**Review by JerryShea — APPROVED** [review]
approved with minor comment

**Pull request merged**